core/templates/dev/head/components/subtitled_html_editor_directive.html

```diff
@@ -0,0 +1,3 @@
+<audio-translations-editor on-start-edit="onAudioTranslationsStartEditAction"
```

Comment:
Hi @jervis446 , I think this is where the core of the problem is. AudioTranslationEditor expects subtitledHtml to be passed in here as a parameter (`subtitled-html="..."`), but you're not doing that, therefore it is undefined and causing the error.

Reply:
OK i got it